example/main.dart

```diff
@@ -8,6 +8,9 @@ class Credentials extends Equatable {
 
   @override
   List<Object> get props => [username, password];
+
+  @override
+  bool get stringable => false;
```

Comment:
Thoughts? I feel like stringify is a bit better because you're specifying you want the Equatable value to be stringified rather than specifying that it is able to be converted to a string.

```suggestion
  bool get stringify => false;
```